packages/kbn-optimizer/src/optimizer/get_mtimes.ts

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 
-import Fs from 'fs';
+import fs from 'fs';
```

Comment:
Please resist the urge to rename like this, I didn't use `Fs` by mistake and it's consistent with the code around it.